.github/workflows/ci-dotnet-action.yml

```diff
@@ -0,0 +1,25 @@
+name: .NET
+
+on:
+  push:
+    branches:
+     - main
+  pull_request:
+    branches:
+     - main
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v3
+      with:
+        dotnet-version: 6.0.x
+    - name: Restore dependencies
+      run: dotnet restore ./backend/Car-Detailing-Backend
```

Comment:
In order for the CI pipelin for .Net to work correctly, the path where the configuration file is located must be specified. If this is not there, GitHub Actions will not work properly because it will not find the path. I have tested various configurations and this is the most optimal one. You can check the logs to verify my words